en/sensor/pmw3901.md

```diff
@@ -0,0 +1,76 @@
+
+The PMW3901 driver was written to be directly plugged into the SPI port on the Pixhawk 4 using the chip select 1. The support for this driver on other boards can then easily be exteded to other boards by modifying the SPI_BUS and the SPI_DEV in the PMW3901 driver. The following diagram shows how to wire the sensor to a Pixhawk 4.
+
+![Bitcraze PH4 Pinout](../../assets/hardware/sensors/pmw3901/ph4-bitcraze-flow-pinout.png)
+
+When plugged into the SPI port of the Pixhawk 4, the Bitcraze flow module will be automatically detected at bootup. No parameters will have to configured, unless you decide to change the orientation of the sensor. This will be explained later in this section together with some diagrams to show the default orientation of the sensor.
+
+### Tindie
+
+The Tindie optical flow module exposes the SPI interface from the PMW3901 module exactly as on the Bitcraze module. The only difference here is the size of the board and the fact that this module doesn't have a distance sensor onboard. Unlike the Bitcraze module where you could try to use the onboard distance sensor, with this one it is mandatory to use one of the [distance sensors](../sensor/rangefinders.md) supported by PX4. 
+
+![Tindie PH4 Pinout](../../assets/hardware/sensors/pmw3901/ph4-tindie-flow-pinout.png)
+
+### Hex
+
+The Hex optical flow module is a very small board containing the PMW3901 flow module, the VL53L1X distance sensor and an IMU to better synchronize the flow data with the gyro data. These three sensors are then sampled by a microcontroller onboard. The goal of the microcontroller is to then expose 2 UAVCAN messages containing all the necessary data.
+
+This board can be connected to the CAN port on any Pixhawk board. All tests were performed connecting it to the CAN1 port on a Pixhawk 4. Differently from the other sensors, this one needs UAVCAN running on PX4. To enable UAVCAN you will have to set the UAVCAN_ENABLE parameter to 2 (if you have have UAVCAN sensors but no motor controllers) or to 3 (if you have also your motor controllers on CAN).
+
+As for the other optical flow boards, it is recommended to use an external distance sensor instead of the onboard one due to the limited range of the VL53L1X. 
+
+![Hex PH4 Pinout](../../assets/hardware/sensors/pmw3901/ph4-hex-optical-flow.png)
+TODO
+
+Regarding rotations, also this board will have a default orientation and if necessary this could be modified through a parameter.
+
+### Thone
+WORK IN PROGRESS
+
```

Comment:
I've also tested this board, the Alientek atk-PMW3901 https://www.aliexpress.com/i/32979605707.html
Had to reverse-engineer some pinouts, only trust the SPI pinouts and V+/GND

![image](https://user-images.githubusercontent.com/1746276/63265767-a459bd80-c28e-11e9-9ce0-e8cc49d09cfd.png)

Reply:
@jkflying Further down your linked page they had this:
![image](https://user-images.githubusercontent.com/5368500/65568595-ca653280-df9d-11e9-97fd-43890fec4d2b.png)

Does that help you with the missing pins?

Reply:
Those are the pins on the parent board this is mounted to, and reflect the labels of the schematic/microcontroller on the other side. There are a bunch of CS lines for example, I had to trace the circuit board to figure out which one is the real one on this board.

Reply:
Thanks. I've sorted by just showing the SPI pins in the connection diagram in other doc. Have included your doc too "for the moment".